_sass/_components/_logo-bar.scss

```diff
@@ -86,6 +65,8 @@ $logo-bar-scale: 14px;
       text-decoration: none;
       font-size: 1rem;
       justify-content: center;
+      min-width: 25%;
+      transition: margin-left 750ms;
 
       &[href*="tracer"],
       &[href*="dropwizard"] {
```

Comment:
Q: as I understand, both "tracer" and "dropwizard" are included with `icon: false`, i.e. as plain text. Is it possible to change the generation script to use a generic name in this case, like "plain-text-logo", so that when new participants are added we do not have to keep changing the css?